scripts/deploy/deploy-memorypipeline.sh

```diff
@@ -0,0 +1,94 @@
+﻿#!/bin/bash
```

Comment:
I would rename all the files with "memorypipeline" to use "memoryservice" instead.  You are not deploying a memory pipeline; you are using these files to deploy a memory service.  We already know that all files here are for pipelines...

Reply:
Same reason as above